bin/fetch-recent-miner-measurements.js

```diff
@@ -218,18 +222,21 @@ async function processRound (roundIndex, measurementCids, resultCounts) {
     prepareProviderRetrievalResultStats: async () => {}
   })
 
+  const isAccepted = (/** @type {Measurement} */ m) =>
```

Comment:
Can this be shared with other parts of the repository? Or is this check only useful in this place?

Reply:
I was thinking about this some more and came to the conclusion that it was a mistake to add committee-based evaluation outcomes to the "fraudAssessment" enum. What I would like to do instead is to have two fields in each measurement:
- `fraudAssessment` based on the tasking algorithm - we can rename it to `taskingEvaluation`
-  `majorityEvaluation` based on the committee & majority algorithm
- rework the code calculating participant scores to look at both `fraudAssessment` and `taskingEvaluation`.

That would prevent mistakes like the one I am fixing in this pull request.

Reply:
That's a great idea 👍

Reply:
That sounds great to me!